test/cmd/apply.sh

```diff
@@ -144,6 +144,14 @@ __EOF__
   output_message=$(! kubectl get pods a 2>&1 "${kube_flags[@]:?}")
   kube::test::if_has_string "${output_message}" 'pods "a" not found'
 
+  # apply b with namespace
```

Comment:
Before running this test, you need to do a clean up.

Then you apply a with the namespace.
Then you apply --prune --namespace <namespace> b
Then you check b exists, but a doesn't

Reply:
@Liujingfang1 
Can you take another look ?

Thanks